examples/medplum-nextjs-demo/app/root.tsx

```diff
@@ -0,0 +1,24 @@
+'use client';
+
+import { MantineProvider } from '@mantine/core';
+import { MedplumClient } from '@medplum/core';
+import { MedplumProvider } from '@medplum/react';
+
+const medplum = new MedplumClient({
+  // Uncomment this to run against the server on your localhost
+  // baseUrl: 'http://localhost:8103/',
+
+  // Handle unauthenticated requests
+  onUnauthenticated: () => (window.location.href = '/'),
+
+  // Use Next.js fetch
+  fetch: (url: string, options?: any) => fetch(url, options),
```

Comment:
I think we might need to test this a little, Next has done some monkey patching around `fetch` in Next 13+ https://nextjs.org/docs/app/api-reference/functions/fetch#:~:text=Next.js%20extends,persistent%20caching%20semantics.

So we should make sure everything works properly

Reply:
I don't know anything about Next.js 14 or current state, but at least at the time this was authored, this line was necessary to take advantage of the Next.js fetch monkey patch.  If this line does not exist, then `MedplumClient` tries to use the global fetch, which does not work (did not work?) with Next.